tools/dockerfile/interoptest/grpc_interop_java/Dockerfile

```diff
@@ -18,9 +18,17 @@ FROM debian:jessie
 RUN echo "deb http://archive.debian.org/debian/ jessie-backports main contrib non-free" > /etc/apt/sources.list.d/jessie-backports.list && \
     echo 'Acquire::Check-Valid-Until no;' > /etc/apt/apt.conf.d/99no-check-valid-until && \
     apt-get update && \
+    apt-get install -y --no-install-recommends curl && \
     apt-get install -y --no-install-recommends -t jessie-backports openjdk-8-jdk-headless && \
     apt-get clean
 
+# Trigger download of as many artifacts as possible, to reduce download
+# flakiness rate. https://github.com/grpc/grpc/issues/18892
```

Comment:
Thought I commented earlier but I didn't see it.

Suggest just adding comment to describe behavior here with respect to:
Will any of the dependencies installed here ever be refreshed during Java' per-interop-run `build_interop.sh`? And if any, which ones?

Reply:
"all the ones that are different" have to be downloaded. It is because the version of grpc-java here and the version during the actual build_interop.sh are unrelated. So it's sort of ¯\_(ツ)_/¯ and hope that it helps. It's not clear what the form of that comment would look like to be helpful.

Reply:
Got it. In that case, maybe add an actionable error message to Java's `build_interop.sh`?

Something like:

```
function die {
  echo $1
  exit 1
}

./gradlew --no-daemon :grpc-interop-testing:installDist -PskipCodegen=true -PskipAndroid=true || die "if gradle failed due to a download failure, re-building the base docker image might help by refreshing cached dependencies"
```